packages/core/src/utils/pretty.ts

```diff
@@ -0,0 +1,34 @@
+import { CoinPretty, PricePretty, RatePretty } from "@keplr-wallet/unit";
+
+export function getPrettyFiat(value?: PricePretty) {
+    if (value) {
+        return value
+            .trim(true)
+            .maxDecimals(2)
+            .toString();
+    } else {
+        return `${value!.symbol}-`;
```

Comment:
```js
if (value) {
  // value is defined
} else {
  // value is undefined here, so value.symbol will never exist
}
```

Reply:
this file is deleted now. it uses `@keplr-wallet/unit`, and these functions will only be used when balance is imported